<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Li Lab</title>
    <link rel="LiLab Icon" type="image/x-icon" href="img/home/LiLab_icon.ico" />
    <link rel="stylesheet" type="text/css" href="css/home.css" />
    <link rel="stylesheet" type="text/css" href="css/tools.css" />
</head>
<body>
    <div id="header">
        <a id="top-logo" href="./index.html">
            <span>Zea&ensp;Systems&ensp;Biology&ensp;Lab</span>
        </a>
        <div id="top-navi">
            <a class="top-navigator" href="./index.html">
                <span>Home</span>
            </a>
            <a class="top-navigator" href="./research.html">
                <span>Research</span>
            </a>
            <a class="top-navigator" href="./publication.html">
                <span>Publication</span>
            </a>
            <div class="top-navigator menu-bar">
                <span>Resource</span>
                <div class="sub-menu">
                    <a class="menu-option" href="./dataset.html">data set</a>
                    <a class="menu-option" href="./tools.html">tools</a>
                </div>
            </div>
            <a class="top-navigator" href="./people.html">
                <span>People</span>
            </a>
        </div>
    </div>

    <div id="content">
        <div class="software-block">
            <div class="software">
                <div class="software-name">DeepBSA</div>
                <div class="software-category">--software--</div>
                <div class="software-brief">
                    We developed a BSA method driven by deep learning (DL) 
                    —DeepBSA for QTL mapping and functional gene cloning, which 
                    is compatible with a variable number of bulked pools and 
                    performed well in both animals and plants. A user-friendly 
                    graphical user interface (GUI) for DeepBSA was produced, 
                    which integrated five widely used BSA algorithms and our 
                    two newly developed algorithms. The DeepBSA software is 
                    freely available to non-commercial users at 
                    http://zeasystemsbio.hzau.edu.cn/tools.html and 
                    https://github.com/lizhao007/DeepBSA.
                </div>
                <div class="software-category" style="margin: 10px 0px 0px;">--cite--</div>
                <div class="software-brief" style="text-align: center;">
                    Li Z., Chen X., Shi S., Zhang H., Wang X., Chen H., Li W., 
                    and Li L. (2022). DeepBSA: A deep-learning algorithm 
                    improves bulked segregant analysis for dissecting complex 
                    traits. Mol. Plant. 
                    doi: 
                    <a href="https://doi.org/10.1016/j.molp.2022.08.004" style="color: black;">
                        https://doi.org/10.1016/j.molp.2022.08.004
                    </a>
                </div>
                <div class="software-category" style="margin: 10px 0px 0px;">--version--</div>
                <div class="software-brief" style="text-align: center;">
                    2022.07.25 version1.1
                </div>
                <div class="software-brief" style="text-align: center;">
                    2022.08.16 version1.2
                </div>
                <div class="software-brief" style="text-align: center;">
                    2022.08.30 version1.3
                </div>
                <div class="software-brief" style="text-align: center;">
                    2022.11.15 version1.4
                </div>
                <a class="software-link" href="/Tools/DeepBSA_windows_v1.4.zip" download>
                    <svg t="1668497158693" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="6996" width="20" height="20">
                        <path d="M523.818667 877.994667l433.962666 70.826666v-400.64H523.818667v329.813334z m0-730.794667v328.704h433.962666V75.093333L523.818667 147.2zM65.109333 475.818667h366.890667V159.104l-366.933333 60.885333v255.829334z m0 330.496l366.890667 59.776v-317.866667h-366.933333v258.090667z" fill="#008000" opacity=".65" p-id="6997"></path>
                    </svg>
                    &nbsp;&thinsp;Download for Windows v1.4
                </a>
                <a class="software-link" href="/Tools/DeepBSA_linux_v1.4.tar.gz" download>
                    <svg t="1668496820479" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="5756" width="25" height="25">
                        <path d="M833.828 772.167c-14.708-27.427-28.724-65.173-32.06-95.097-7.064-63.345 7.001-95.581-26.311-157.655-20.001-37.27-47.167-69.818-69.762-105.374-57.008-89.708-35.862-181.666-78.106-273.828-14.677-32.021-39.272-58.288-73.372-69.27-40.023-12.889-88.039-6.905-123.821 15.09-97.955 60.213-26.601 240.65-82.234 332.558-34.997 57.817-72.677 108.993-92.906 175.374-15.962 52.376-11.905 116.451-74.135 139.232-16.782 6.143-41.737 5.268-50.966 23.624-5.79 11.518-1.802 24.689 0.951 36.449 2.356 10.066 3.024 19.861 1.87 30.14-1.651 14.71-7.417 28.869-8.274 43.641-0.878 15.142 8.565 24.551 21.704 30.534 47.398 21.585 113.582 34.279 164.669 43.764 66.028 12.258 59.608-7.377 115.821-23.029 39.692-11.052 88.466-13.955 129.206-6.827 43.073 7.536 72.966 44.046 112.09 48.73 33.22 0 210.129-91.293 212.144-131.077 1.811-35.802-33.736-33.163-46.508-56.979zM561.696 188.612c34.429 24.892-1.788 99.81-14.799 87.114 41.625-32.641-34.982-83.122-29.425-11.704-53.249-11.693-3.247-109.731 44.224-75.41z m-4.039 117.923c1.857 28.37-64.491 48.349-85.296 56.455-79.455-0.65-53.966-82.141 7.617-88.858 11.889-1.296 76.451 13.646 77.679 32.403zM407.938 198.841c42.337-29.134 70.265 43.258 33.357 66.249 16.796-88.043-79.787-11.981-12.48 11.418-26.974 29.286-46.681-59.91-20.877-77.667z m-57.874 731.028c-38.514-10.84-181.601-28.379-200.949-63.937-0.276-0.507-0.351-2.272-0.282-4.883-0.019-0.032-0.042-0.063-0.06-0.095-0.833-1.415-0.113-8.165 1.246-16.547 1.879-19.977 5.394-46.894 6.514-49.347 12.905-28.267 12.771-23.836 21.86-26.291 5.612-3.079 13.887-7.607 27.549-19.429 37.781-32.691 49.648-51.05 71.076-37.382 8.427 2.949 16.297 11.207 25.986 25.067a570.9 570.9 0 0 1 8.53 10.204c16.954 20.727 55.205 71.093 68.786 114.074 13.447 30 13.992 58.501-30.256 68.566z m290.721-232.992c-16.92 43.195-1.196 82.899-35.488 120.646-58.579 64.48-168.292 61.287-220.377 1.598 61.991-24.918-79.397-135.334-84.198-181.037-10.461-99.563 81.484-201.457 102.457-296.773 77.359 82.013 125.198-33.75 189.934 21.718 50.436 43.215 102.409 212.603 87.712 277.829-8.345 37.034-32.364 36.425-40.04 56.019z m103.371 199.109c-159.991 95.734-70.13-147.276-65.785-178.44 11.336 103.265 74.062 18.695 110.579 7.453-17.789 5.477 57.616 96.436 56.788 86.041 3.123 39.223-63.176 61.965-101.582 84.946z" p-id="5757" fill="#008000"></path>
                    </svg>
                    &ensp;Download for Linux v1.4
                </a>

            </div>
            <div class="software">
                <div class="software-name">iBP Genotyping</div>
                <div class="software-category">--web service--</div>
                <div class="software-brief">
                    A cost-saving novel genotyping and epigenotyping method 
                    for Next-Generation Sequencing (NGS) data. On this basis, 
                    the website provides convenient analysis functions for the 
                    NGS data obtained in the experiment, making genotyping 
                    easier.
                </div>
                <a class="software-link"  href="http://zeasystemsbio.hzau.edu.cn/tools/ibp" target="_blank">
                    http://zeasystemsbio.hzau.edu.cn/tools/ibp
                </a>
            </div>
            <div class="software">
                <div class="software-name">Maize Interactome</div>
                <div class="software-category">--database--</div>
                <div class="software-brief">
                    The first-generation multi-omics integrative functional map 
                    of maize, which contains over 15 million edges for more 
                    than 1000 functional modules, covering the whole genetic 
                    transmission from genome, to transcriptome, to translatome, 
                    and to proteome.
                </div>
                <a class="software-link" href="http://minteractome.ncpgr.cn/index.php" target="_blank">
                    http://minteractome.ncpgr.cn/index.php
                </a>
            </div>
        </div>
    </div>

    <div id="footer">
        <div id="info">
            <span>地址：湖北省武汉市洪山区狮子山街1号</span>
            <span>邮编：430070</span>
            <span>Copyright &copy; <span id="year"></span> Li Lab</span>
        </div>
        <div id="record">
            <span>鄂ICP备13013419号</span>
            <span>鄂公网安备 42011102000808号</span>
        </div>
    </div>
    <script type="text/javascript" src="js/lilab_website.js"></script>
</body>
</html>